packages/deployer/subtasks/validate-router.js

```diff
@@ -20,36 +11,50 @@ subtask(
 ).setAction(async (_, hre) => {
   logger.subtitle('Validating router');
 
-  const { contracts } = await getSourcesAST(hre);
-  const modules = filterValues(hre.deployer.deployment.general.contracts, (c) => c.isModule);
+  const sourceErrorsFound = await _runSourceValidations();
```

Comment:
Who needs comments when a function name explains what a contained piece of code does?